test/test.js

```diff
@@ -38,5 +39,19 @@ require('../glpk.js').then(glpkjs => {
 		})
 		t.end();
 	});
+  
+  tape('The time limit should kill the solver before finding optimal solution', { timeout: 99999 }, t => {
```

Comment:
Curious: why the large number here for timeout?

Reply:
no reason, I just copy-pasted the other test and I didn't touch it